src/promptflow/promptflow/_sdk/operations/_run_operations.py

```diff
@@ -227,6 +227,22 @@ def update(
         ORMRun.get(name).update(display_name=display_name, description=description, tags=tags, **kwargs)
         return self.get(name)
 
+    @monitor_operation(activity_name="pf.runs.delete", activity_type=ActivityType.PUBLICAPI)
+    def delete(
+        self,
+        name: str,
+    ) -> None:
+        """Delete run permanently.
+
+        :param names: run names to delete
+        :param kwargs: other fields to update, fields not supported will be directly dropped.
+        :return: None
+        """
+        run_name = Run._validate_and_return_run_name(name)
```

Comment:
This line is redundant because it will be validated in the get function.
![image](https://github.com/microsoft/promptflow/assets/23182548/2483aad1-a5d1-4f6c-8673-3ca45d1a4d31)

Reply:
good catch